spec/spec_helper.rb

```diff
@@ -1,10 +1,26 @@
 require "simplecov"
+require 'tmpdir'
 
 SimpleCov.start do
   add_filter "/spec/"
 end
 
 require_relative "../lib/calendar_assistant"
+
+def use_clean_config
+  temp_home_dir = Dir.mktmpdir
+  ENV["CA_HOME"] = temp_home_dir
+
+  yield
+
+  FileUtils.remove_entry temp_home_dir
+end
+
+# CLI::Commands reads local config to generate help and options
+use_clean_config do
+  require_relative "../lib/calendar_assistant/cli"
```

Comment:
This pretty dodgy piece of code is here to help the code be _loaded_ with a clean environment. 

It does not ensure that the code is _executed_ in a clean environment which we probably should do.

Raised #127 to address the local configuration bizzo in tests